types/react/index.d.ts

```diff
@@ -37,6 +37,7 @@
 
 import * as CSS from 'csstype';
 import * as PropTypes from 'prop-types';
+import { Interaction as SchedulerInteraction } from 'scheduler/tracing';
```

Comment:
Is the idea here, unlike with `csstype` where we have to specify the dependency in our `package.json`, that because react uses `scheduler` that any project using the react types will also have `scheduler` in their `node_modules` and therefore we don't need to update our own `package.json` here?  What would be the downside of specifying this.

Apologies if I missed something.

Reply:
type definitions under `@types/*` are added automatically to the package.json. Only packages that bundle their definitions need to be listed in the package.json. That's why we don't list `@types/prop-types` in the package.json.

Reply:
ahh, okey dokey.  thanks for the explanation.